libbeat/processors/add_process_metadata/add_process_metadata.go

```diff
@@ -177,9 +215,24 @@ func (p *addProcessMetadata) enrich(event common.MapStr, pidField string) (resul
 			return nil, err
 		}
 	}
+
 	return result, nil
 }
 
+// add container.id into meta for mapping to pickup
```

Comment:
```suggestion
// enrichContainerID adds container.id into meta for mapping to pickup.
```

I like to follow godoc style even for unexported types/methods.